presto-native-execution/presto_cpp/main/connectors/arrow_flight/tests/data/mtls_certs/ca.crt

```diff
@@ -0,0 +1,33 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
How did you generate the client certs? In the `data` directory there is `generate_tls_certs.sh` script. We should be able to duplicate the certs used here